module/app/upgrades/polaris/README.md

```diff
@@ -0,0 +1,9 @@
+# POLARIS UPGRADE
+
+The *Polaris* upgrade contains the following changes.
+
+## Summary of Changes
+
+* Enable the use of Ethereum keys for Gravity transactions.
+This change enables users to sign Gravity transactions using their Ethereum wallet (e.g. MetaMask).
+Notably, this support enables IBC Auto-Forwarding functionality to work for Cosmos EVM chains (e.g. Evmos) without the potential for loss of funds.
```

Comment:
this could be a little clearer and highlight that funds are placed into a re-prefixed key and since pubkey derivation is different these funds would be irretrievable.